.eslintrc

```diff
@@ -0,0 +1,77 @@
+{
+  // Bable parser required to support the spread operator
+  // in jsx. See: https://github.com/eslint/espree/issues/116
```

Comment:
Yeah - eslint has magic json which allows comments!  It's a bit weird but having comments is useful.